test/TestUtils/TestProjectFixture.cs

```diff
@@ -0,0 +1,290 @@
+﻿using Microsoft.DotNet.Cli.Build;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.DotNet.CoreSetup.Test
+{
+    public class TestProjectFixture
```

Comment:
Why can't this logic exist in the TestProject class?  It wouldn't be duplicated, and may make more sense.

From my understanding a "fixture" is used when you can apply it to multiple different things.  But we only have 1 thing we are applying this to: a TestProject.

---
In reply to: [66107984](https://github.com/dotnet/core-setup/pull/97#discussion_r66107984) [](ancestors = 66107984,66102620)